src/test/regress/citus_tests/test/test_other_databases.py

```diff
@@ -5,6 +5,8 @@ def test_main_commited_outer_not_yet(cluster):
     # create a non-main database
     c.sql("CREATE DATABASE db1")
 
+    c.sql("SELECT citus_set_coordinator_host('localhost')")
+
```

Comment:
Is this actually necessary? Afaict `CitusCluster` its `__init__` function already does this. If that doesn't work for some reason we should fix it there instead of working around the problem here.